migration/faqs.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+#### Does every Nimiq user need to run the migration tools to transition to PoS?
+
+No, participation in the transition process is not mandatory. However, if you want to be one of our first validators for the new PoS chain or migrate the state, you can find the guide that best fits your preferences [here](/migration/).
+
+#### I own a Nimiq Wallet. Do I need to participate in the migration?
+
+No, you don’t need to participate. Your balance and transaction history remains intact during the entire process.
+
+#### Can I access my wallet during the transition?
+
+Yes, you can access your wallet at any time during the transition.
+
+#### Can I log in to the Nimiq Wallet with the same Login File after the transition?
+
+Yes, you can. Since your data remains intact, your Login File will remain the same. In case you log out for any reason, you can easily log back in using the same file and 24 recovery words.
+
+#### Do I need to be a miner to register as a validator?
+
+No, you don't. Any type of user can register as a validator, regardless of whether they are a miner or not.
+
+#### Do I need to register as a validator to participate actively in the migration?
+
+No, registration as a validator is optional. Users can choose to actively participate either as a validator or as an observer. Both options are available but participation in the migration is not mandatory for any Nimiq user.
+
+#### Can I deposit more than the minimum required as a pre-registered validator?
+
+Yes. Any NIM amount exceeding the minimum deposit of 100 000 NIM will be considered stake, but only if it exceeds the minimum stake of 100 NIM. If the excess does not meet this minimum, it will be burned.
+
+#### Will my balance remain intact during the transition?
+
+Yes, your balance and transaction history will remain intact throughout the transition.
+
+#### I don’t want to participate in the transition, but I want to earn rewards. How can I do it?
+
+You can participate in the pre-staking phase by delegating NIM to one of the pre-registered validators. Simply select a validator from the [provided list]() and pre-stake your NIM to earn rewards post-transition.
```

Comment:
I guess provided list is a placeholder right now?

Reply:
yes, once we have the list I'll update the link